mgradm/shared/kubernetes/k3s.go

```diff
@@ -29,7 +29,7 @@ func InstallK3sTraefikConfig(debug bool) {
 }
 
 // RunPgsqlVersionUpgrade perform a PostgreSQL major upgrade.
-func RunPgsqlVersionUpgrade(registry string, image types.ImageFlags, upgradeImage types.ImageFlags, nodeName string, oldPgsql string, newPgsql string) error {
+func RunPgsqlVersionUpgrade(registry string, image types.ImageFlags, upgradeImage types.ImageFlags, namespace string, nodeName string, oldPgsql string, newPgsql string) error {
```

Comment:
Super long lines like this one should be cut like this:

```suggestion
func RunPgsqlVersionUpgrade(
	registry string,
	image types.ImageFlags,
	upgradeImage types.ImageFlags,
	namespace string,
	nodeName string,
	oldPgsql string,
	newPgsql string,
) error {
```